src/api/forms/service.js

```diff
@@ -291,8 +290,15 @@ export async function updateFormMetadata(formId, formUpdate, author) {
 
     return updatedForm.slug ?? form.slug
   } catch (err) {
+    if (
+      Boom.isBoom(err) &&
+      err.name === 'MongoServerError' &&
```

Comment:
I am leaving this because I think it could be useful info but let me know if you disagree, I can remove it.